src/Compilers/VisualBasic/Test/Symbol/SymbolDisplay/SymbolDisplayTests.vb

```diff
@@ -5014,13 +5014,13 @@ End Class
                 SymbolDisplayPartKind.Space,
                 SymbolDisplayPartKind.Keyword,
                 SymbolDisplayPartKind.Space,
-                SymbolDisplayPartKind.ClassName,
+                SymbolDisplayPartKind.StructName,
                 SymbolDisplayPartKind.Punctuation,
                 SymbolDisplayPartKind.Keyword,
                 SymbolDisplayPartKind.Space,
                 SymbolDisplayPartKind.StructName,
                 SymbolDisplayPartKind.Punctuation},
-                references:={MetadataReference.CreateFromImage(TestResources.NetFX.ValueTuple.tuplelib)})
+                references:={Net461.ExtraReferences.SystemValueTuple})
```

Comment:
>Net461.ExtraReferences.SystemValueTuple [](http://example.com/codeflow?start=29&length=39)

Should we just use the legacy reference here, to avoid changing the expected result?

Reply:
I thought about that but symbol display is a user visible artifact. Decided that it was better to have our tests track down Tuple displays in the 99.999% case vs. the .001% case that I'm not sure you can reaally achieve.